cmake/modules/FindFakeit.cmake

```
@@ -0,0 +1,20 @@
+find_path(Fakeit_ROOT_DIR
+  NAMES include/fakeit.hpp
+)
```

Except for the header file `fakeit.hpp`, I suggest we follow the naming convention for FakeIt and capitalize the 'I'. So **FakeIt** and not _Fakeit_ - both for CMake variables and filenames:

cmake/modules/FindFakeit.cmake - > cmake/modules/FindFakeIt.cmake

and

```suggestion
find_path(FakeIt_ROOT_DIR
  NAMES include/fakeit.hpp
)
```

If you agree, clean up whereever this applies.